1.7/administration/monitoring/view-node-details.md

```diff
@@ -0,0 +1,125 @@
+---
+post_title: Viewing node details
+nav_tile: Node details
+menu_order: 1
+---
+You can get a comprehensive list of the apps being run on your DC/OS cluster nodes.
+
+**Prerequisites:** [DC/OS and DC/OS CLI][1] are installed
+
+1.  SSH into your master node.
+    
+        $ dcos node ssh --leader --master-proxy
+        
+    
+    For more information, the SSH [documentation][2].
+
+2.  Run this command from your master node to view the node details:
```

Comment:
Should we suggest querying through AdminRouter? I wish we had a patched http or curl that added a token.